### DISCUSSION TITLE
Is a LAN-Port or WPA-Enterprise supported

### DISCUSSION THREAD

**Discussion by Bluxomat** [X1Plus/X1Plus#256] [General]
Hi there.
I think about getting a X1C BECAUSE of X1plus.
My question is if an LAN-Port via USB will be supported.
Or at least WPA-Enterprise?

Thanks a lot for running this project!

Cheers
Bluxomat

**Comment by balika011** [comment]
I made a few patches for the Ethernet that works great on my own setup, however Joshua didn't like them, so they didn't even make it into a PR.

**Reply by diginfo** [reply]
@balika011 - Oops, did not see this before I posted. 

Mind sharing what patches you made ?
